frontend/src/reducers/user.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+    login: {
+    accessToken: null,
+    userId: 0,
+    statusMessage: "",
+    isLoggedIn: false
+    },
+};
+
+export const user = createSlice({
+    name: 'user',
+    initialState: initialState,
+    reducers: {
+        setAccessToken: (state, action) => {
+            const { accessToken } = action.payload;
+            state.login.accessToken = accessToken;
+        },
+        setUserId: (state, action) => {
+            const { userId } = action.payload;
+            state.login.userId = userId;
+        },
+        setStatusMessage: (state, action) => {
+            const { statusMessage } = action.payload;
+            state.login.statusMessage = statusMessage;
+        },
+        logout: (state, action) => {
+            state.login.isLoggedIn = false;
+            state.login.userId = 0;
+            state.login.accessToken = null;
+            state.login.statusMessage = "";
+        },
+    } 
+})
```

Comment:
Reducer is a great place to manage your users but you need to store part of data in the local storage to make your users logged in after they refresh 🤓